src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -82,7 +102,16 @@ public String greeting(@RequestParam(name="name", required=false, defaultValue="
      * @throws DatabaseException: When the save method throws an error coming from the database
      */
     @PutMapping("{id}")
-    public ResponseEntity<Book> update(@PathVariable int id,@Valid @RequestBody Book book) throws BookNotFoundException, IdMismatchException, DatabaseException {
+    @ApiOperation(value = "Updates a specified book", response = User.class)
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "OK"),
+            @ApiResponse(code = 400, message = "Bad Request"),
+            @ApiResponse(code = 404, message = "Not Found")
```

Comment:
you can extract the messages to constants

Reply:
Done!